app/forms/csv_upload_form.rb

```diff
@@ -21,7 +21,7 @@ def save(file)
     begin
       Keyword.transaction do
         # rubocop:disable Rails/SkipsModelValidations
-        user.keywords.insert_all parsed_keywords
+        @keyword_ids = user.keywords.insert_all(parsed_keywords).map { |id| id['id'] }
```

Comment:
```
   user.keywords.insert_all(parsed_keywords).map { |keyword| keyword['id'] }
```

Reply:
Here I would not recommend using `keyword` as postgres returns by default an `array of PK`. Naming this array `keyword` exposes us to a wrong assumption that it contains attributes others than `id` (like name, created_at, ...). 

What about `returning_ids` or `insert_response` @ankitkalia1195 ? :)

Reply:
can be `keyword_hash`? 

```
    user.keywords.insert_all(parsed_keywords).map { |keyword_hash| keyword_hash['id'] }
```

Reply:
Let's go for this one! :)